BlakeDonn/ideabox-boilerplate
# Form mobile view

What’s this PR do? 

Adds css properties to the idea-form to replicate project example. These properties alter the following elements: text title/input, body title/input, save-idea button, and the search idea button/input.
​

Where should the reviewer start?

Should start at index.html to make sure no merge conflicts arise, I believe only the relevant elements were altered but could have had a typo. Should then check css from line 89-down.

How should this be manually tested?

Open index.html to view the idea-form in mobile view, and change sizes to ensure everything is responding correctly.
​

Any background context you would like to provide?

The use of em values will most likely need to be changed for many of these to ensure responsiveness once changing to desktop view and in-between. Font styles and sizes will need to be adjusted, as well as refined padding.
​

What are the relevant tickets?

Next steps will to divide and conquer for all sections in larger breakpoints.

Screenshots (if appropriate): 

![idea-form](https://i.imgur.com/3YsOa7v.png)
